docs/Tutorial_VALMOD.ipynb

```diff
@@ -0,0 +1,688 @@
+{
```

Comment:
<p>"distance" rather than "distane" (missing a "c")</p><p> </p><p>What is "inequation"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/586/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
I assume the comment is for the previous version of notebook. So, I ignore this. If I am not correct, please let me know.